pkg/controller/operandregistry/operandregistry_controller.go

```diff
@@ -100,24 +99,20 @@ type ReconcileOperandRegistry struct {
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
 func (r *ReconcileOperandRegistry) Reconcile(request reconcile.Request) (reconcile.Result, error) {
 	// Commented out unused logging
-	// reqLogger := log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
-	// reqLogger.Info("Reconciling OperandRegistry")
+	reqLogger := log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
```

Comment:
The reason why I comment this loger out is this controller doesn't reconcile the resource. it just initialize OperandRegistry instance. If you think we need to show the logs in this controller. I think we should update 
```
	reqLogger.Info("Reconciling OperandRegistry")
```

to

```
	reqLogger.Info("Initializing OperandRegistry instance")
```